php/deepgram-transcribe-video/README.md

```diff
@@ -0,0 +1,66 @@
+
+# ⚡ deepgramTranscribeVideo()
+
+A Php Cloud Function for transcribing video using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+    {
+
+        "fileUrl":"https://rawcdn.githack.com/deepgram-devs/transcribe-videos/62fc7769d6e2bf38e420ee5224060922af4546f7/deepgram.mp4"
+
+    }
+}
+```
+
+_Successful function response::_
+
+```json
+{ "deepgramData": {}, "success": true }
+```
+
+_Error function response:_
+
+```json
+{ "message": "error message", "success": false }
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Please remove references to "environment" as they should only be referred to as variables.